src/main/java/org/asciidoctor/maven/AsciidoctorMojo.java

```diff
@@ -242,10 +319,9 @@ private void setDestinationPaths(OptionsBuilder optionsBuilder, final File sourc
 
     protected Asciidoctor getAsciidoctorInstance(String gemPath) throws MojoExecutionException {
         Asciidoctor asciidoctor = null;
-        if (gemPath == null) {
-             asciidoctor = Asciidoctor.Factory.create();
-        }
-        else {
+        if (gemPath == null || gemPath.isEmpty()) {
```

Comment:
I think a value of empty string is supposed to be passed to `create()`. This ensures that Asciidoctor is isolated.